src/main/java/Duke.java

```diff
@@ -1,10 +1,217 @@
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static final String intro = "Hi! I'm Duke! :)\nHow may I help?";
+    private static final String outro = "Goodbye!";
+    private static ArrayList<Task> list = new ArrayList<>();
+
+    private static void reset() {
+        System.out.println();
+        System.out.print("User: ");
     }
+    
+    private static void addToList(Task t) {
+        list.add(t);
+        System.out.print("Duke: ");
+        System.out.println("added " + t.getTask() + "!");
+        System.out.println("Now you have " + list.size() + " tasks in the list.");
+    }
+
+    private static void emptyErr() {
+        System.out.println("Duke: Error!! List empty! Try again.");
+        reset();
+    }
+
+    private static void viewList() {
+        System.out.println("Here's your list of tasks:");
+        System.out.println();
+        for(int i  = 0; i < list.size(); i++) {
+        System.out.println(i+1 + ". " + list.get(i));
+        }
+    }
+
+    private static void markTask(int num) {
+        if(num >= list.size()) {
+            System.out.println("Duke: Task no." + (num+1) + " not found. Try again.");
+        } else {
+            list.get(num).mark();
+            System.out.println("Duke: Nice It's marked!");
+            System.out.println(list.get(num));
+        }
+    }
+
+    private static void unmarkTask(int num) {
+        if(num >= list.size()) {
+            System.out.println("Duke: Task no." + (num+1) + " not found. Try again.");
+        } else {
+            list.get(num).unmark();
+            System.out.println("Duke: Ok! It's unmarked!");
+            System.out.println(list.get(num));
+        }
+    }
+
+    private static ToDo makeToDo(String[] task) {
+        StringBuilder sb = new StringBuilder();
+        for(int i = 1; i < task.length; i++) {
+            sb.append(task[i] + " ");
+        }
+        return new ToDo(sb.toString());
+    }
+
+    private static Event makeEvent(String[] task) throws DateTimeParseException {
+        StringBuilder start = new StringBuilder();
+        StringBuilder end = new StringBuilder();
+        StringBuilder desc = new StringBuilder();
+        boolean isDescripton = true;
+        boolean first = false;
+        boolean second = false;
+
+        for(int i = 1; i < task.length; i++) {
+            if(task[i].equalsIgnoreCase("/from")) {
+                first = true;
+                isDescripton = false;
+                continue;
+            }
+            if(task[i].equalsIgnoreCase("/to")) {
+                second = true;
+                first = false;
+                continue;
+            }
+            if(first) {
+                start.append(task[i] + " ");
+            }
+            if(second) {
+                end.append(task[i] + " ");
+            }
+            if(isDescripton) {
```

Comment:
Spacing can be added to after each `if` to follow the coding standards
```suggestion
            if (task[i].equalsIgnoreCase("/from")) {
                first = true;
                isDescripton = false;
                continue;
            }
            if (task[i].equalsIgnoreCase("/to")) {
                second = true;
                first = false;
                continue;
            }
            if (first) {
                start.append(task[i] + " ");
            }
            if (second) {
                end.append(task[i] + " ");
            }
            if (isDescripton) {
```